recipe/meta.yaml

```diff
@@ -67,3 +64,4 @@ extra:
   recipe-maintainers:
     - primozgodec
     - mariusvniekerk
+    - xylar
```

Comment:
I certainly don't need to be a maintainer but I thought it would be friendly to add myself in case I can be helpful.